gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, conf, controller):
         self._controller = controller
         self.rpc_event_mapping = {
                           'healthmonitor': ['DEVICE_HEALTHY',
+                                           'DEVICE_NOT_REACHABLE',
```

Comment:
This is needed for error scenario, where in create path healthmonitor is not present, as 2nd element of list gets accessed in error scenario.
We observed this issue while testing.